src/assets/styles/mixins/cards.scss

```diff
@@ -33,4 +33,11 @@
     margin: 0;
     padding: 15px;
   }
+
+  a {
```

Comment:
it may affect other buttons:
<img width="371" alt="image" src="https://user-images.githubusercontent.com/351613/174091220-d33e8bb4-342e-4599-a59e-1ba3ff6f17e8.png">